packages/textmate-grammars/src/browser/perl.ts

```diff
@@ -22,6 +22,30 @@ export class PerlContribution implements LanguageGrammarDefinitionContribution {
 
     readonly id = 'perl';
     readonly scopeName = 'source.perl';
+    readonly config: monaco.languages.LanguageConfiguration = {
```

Comment:
https://github.com/Microsoft/vscode/blob/master/extensions/perl/perl.language-configuration.json - config which used in VS Code

Reply:
Hi @akosyakov 

In this case I actually copied it from the Theia repo.

https://github.com/theia-ide/theia/blob/master/packages/textmate-grammars/src/browser/python.ts#L23

Do you need meed to change anything?

Reply:
I would expect that we align grammars with VS Code

Reply:
Ok, I copied it, I did have to add open and close because it was required for the pair types.
Appveryor is not passing, but I think that is a problem related to the Java LS?

Reply:
yes, windows build is filing sometimes

Reply:
yes, windows build is filing sometimes